linera-execution/src/lib.rs

```diff
@@ -345,12 +345,26 @@ pub enum ExecutionError {
 impl From<ViewError> for ExecutionError {
     fn from(error: ViewError) -> Self {
         match error {
+            ViewError::InactiveChain(chain_id) => ExecutionError::InactiveChain(chain_id),
             ViewError::BlobsNotFound(blob_ids) => ExecutionError::BlobsNotFound(blob_ids),
```

Comment:
I know this is old tech debt but can we move these errors out of `ViewError`?

Reply:
Also this PR is doing many things and we may want to use several of them instead.

Reply:
What I propose is to merge this PR and then pay back the tech debt in a subsequent PR the problem by introducing `StorageError`.

The change would likely be a large one.